app/js/checkAnswers.js

```diff
@@ -129,3 +123,56 @@ function displayResult(earnedPoints, earnedPercentage, answeredQuestions) {
     document.querySelector(".answered_questions").innerHTML = answeredQuestions
     document.querySelector(".score_percentage").innerHTML = earnedPercentage
 }
+
+
+function questionAnswered() {
+    let answers = getUserAnswers(30)
```

Comment:
Hard coding this is not a great solution as the number of questions may change.  Could check number of divs with class of 'question'.

Reply:
would require total refactor as they have used the constant questionAmount = 30 everywhere.
but i made my functions dynamic. :)